draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2836,28 +3039,27 @@ https://video.example.com/k/movie1/4/013.ts
 
        
 
-            <section title="CI/T Playlist Protocol Capability Object">
+            <section title="CI/T Object List Type Capability Object">
         <t>
-           Given a content-playlist being supported by the dCDN, the CI/T Playlist Protocol capability object is used to indicate support for one or more
-           MediaProtocol types listed in <xref target="IANA.CDNI.MediaProtocolReg"/> by the playlists
-           property of the "trigger.v2" object. Capability type is  "FCI.TriggerSpecPlaylistProtocol".
+           Given an object list being supported by the dCDN, the CI/T Object List Type capability object is used to indicate support for one or more
+           Object List types listed in <xref target="IANA.CDNI.ObjectListTypeReg"/> by the type property of the "ObjectList" object. Capability type is  "FCI.TriggerObjectListType".
```

Comment:
Nit:  While I realize the original source had this small error (and we didn't catch it because it occurred at a line break), it would be good to eliminate the extra space after "Capability type is  ".